fire/cli/info.py

```diff
@@ -382,25 +384,52 @@ def punkt(ident: str, obs: str, koord: str, detaljeret: bool, **kwargs) -> None:
 
 
 @info.command()
+@click.option(
+    "--alle", is_flag=True, default=False, help="Udskriv også tidsserier",
+)
 @fire.cli.default_options()
-@click.argument("srid")
-def srid(srid: str, **kwargs):
+@click.argument("srid", required=False)
+def srid(srid: str, alle: bool, **kwargs):
     """
     Information om et givent SRID (Spatial Reference ID)
 
     Eksempler på SRID'er: EPSG:25832, DK:SYS34, TS:81013
+
+    Anføres ikke SRID gives liste af mulige SRID. Som standard uden lokale
+    tidsseriekoordinatsystemer
+
+    tilvalg ``--alle` kan kun vælges uden angiven SRID. Udvider listen med 
+    lokale tidsseriekoordinatsystemer
     """
-    srid_name = srid
+    if not srid:
+        srid_db = firedb.session.query(Srid).all()
```

Comment:
Hvis du filtrerer SRID'erne direkte på databasen kan du spare dig selv for en del besvær længere nede (se næste kommentar)
```suggestion
        if alle:
            srid_db = firedb.session.query(Srid).order_by(Srid.name).all()
        else:
            srid_db = (
                firedb.session.query(Srid)
                .filter(not_(Srid.name.like("TS:%")))
                .order_by(Srid.name)
                .all()
            )
```